### ISSUE TITLE
What is the recommended camera model / spec?

### ISSUE THREAD

**Issue by nuntipat** [zju3dv/EasyMocap#7]
Hi, May I know the model of the camera you use to prepare the sample dataset? Does it have to be genlock or we can just synchronize it manually using clapperboard? I'm planning to try your code on a set of IP cameras but I'd love to hear you though on this setup. Thank you.

**Comment by chingswy** [comment]
Our sample dataset is synchronized by an external trigger. If your cameras do not support this, you can synchronize them manually using a clapperboard or the timestamp. 
Hope you can share your experience if you succeed, which is very helpful for others: )

**Comment by carlosedubarreto** [comment]
The experience I had was a mixed of failure and success.

At first I tryed something to synchonize the recording start and finish. but i didnt work well on slower PCs, so I ended up using  some sort of "clapperboard", my shoes.

the synchronized one that I  did was using OBSStudio and the Droidcam OBS studio app.
On obs studio I made the screen the size to put all the cameras, I tested with 4 cameras, so I had one camera in each corner of the screen to record.

the "shoes clapperboard" is did this way. I manually started recordong each phone and whe I started the last one, I went to the center of the screen and released my shoe.
To sync them was really easy 😊

the example
![image](https://user-images.githubusercontent.com/4061130/111121421-62b4f580-854b-11eb-8621-55e35522d8b1.png)
